src/components/namestable/worker-edit.component.tsx

```diff
@@ -23,7 +23,7 @@ import { useDispatch } from "react-redux";
 
 const useStyles = makeStyles({
   container: {
-    height: "100%",
+    height: "93%",
```

Comment:
to jest po to, żeby elementy na dole drawera nadal były widoczne po tym, jak dodałam większy margines u góry